test/benchmarks/src/main/scala/scala/collection/mutable/OpenHashMapRunner.scala

```diff
@@ -0,0 +1,111 @@
+package scala.collection.mutable
+
+import java.io.BufferedWriter
+import java.io.File
+import java.io.FileOutputStream
+import java.io.OutputStreamWriter
+import java.io.PrintWriter
+import scala.collection.JavaConversions
+import scala.language.existentials
+import org.openjdk.jmh.results.RunResult
+import org.openjdk.jmh.runner.Runner
+import org.openjdk.jmh.runner.options.CommandLineOptions
+import org.openjdk.jmh.runner.options.Options
+import benchmark.JmhRunner
+import org.openjdk.jmh.runner.options.OptionsBuilder
+import org.openjdk.jmh.runner.options.VerboseMode
+
+/** Replacement JMH application that runs the [[OpenHashMap]] benchmark.
+  * 
+  * Outputs the results in a form consumable by a Gnuplot script.
+  */
+object OpenHashMapRunner extends JmhRunner {
+  /** File that will be created for the output data set. */
+  private[this] val outputFile = new File(outputDirectory, "OpenHashMap.dat")
+
+  /** Qualifier to add to the name of a memory usage data set. */
+  private[this] val memoryDatasetQualifer = " memory"
+
+  /** Name of the JMH parameter for the number of map entries per invocation. */
+  private[this] val sizeParamName = "size"
+
+  /** Name of the JMH auxiliary counter that collects operation counts. */
+  private[this] val operationsAuxCounterName = "operations"
+
+  /** Name of the JMH auxiliary counter that collects memory usage. */
+  private[this] val memoryAuxCounterName = "memory"
+
+  /** Name of the JMH auxiliary counter that collects the number of map entries. */
+  private[this] val entriesAuxCounterName = "mapEntries"
+
+  def main(args: Array[String]) {
+    import scala.collection.JavaConversions._
+    import scala.language.existentials
+  
+    val opts = new CommandLineOptions(args: _*)
+    var builder = new OptionsBuilder().parent(opts)
+      .jvmArgsPrepend("-Xmx6000m")
+    if (!opts.verbosity.hasValue)  builder = builder.verbosity(VerboseMode.SILENT)
+
+    val results = new Runner(builder.build).run()
+
+    // Sort the results
+
+    /** Map from data set name to data set. */
+    val datasetByName = Map.empty[String, Set[RunResult]]
+
+    /** Ordering for the results within a data set. Orders by increasing number of map entries. */
+    val ordering = Ordering.by[RunResult, Int](_.getParams.getParam(sizeParamName).toInt)
+
+    def addToDataset(result: RunResult, key: String): Unit =
+      datasetByName.get(key)
+        .getOrElse({ val d = SortedSet.empty(ordering); datasetByName.put(key, d); d }) += result
```

Comment:
I generally prefer dot-on-previous-line because it works better when e.g. pasting into the REPL.  The indentation is already plenty of indication that it's a continuation of the previous command.  Also, the `+= result` bit is so far to the right it's easy to miss.  I'd just assign a temporary variable.  And are you not using `getOrElseUpdate` for some reason?  That might make it short enough so a single line is fine.